Sources/Workspace/UserToolchain.swift

```diff
@@ -223,12 +223,12 @@ public final class UserToolchain: Toolchain {
 
     /// Returns the path to llvm-cov tool.
     public func getLLVMCov() throws -> AbsolutePath {
-        return try UserToolchain.getTool("llvm-cov", binDir: self.destination.binDir)
+        return try UserToolchain.getTool("llvm-cov", binDir: self.swiftCompilerPath.parentDirectory)
```

Comment:
This one and the one below were actually wrong and didn't work in debug builds of SwiftPM, as these tools come from the toolchain and are not built by SwiftPM.  The new code is the same as for the symbol graph extraction tool and the other similar tools.